test/bin/run_plugin.spec.js

```diff
@@ -43,19 +40,16 @@ describe('Run plugin CLI', function () {
     });
 
     after(function (done) {
-        testFixture.forceDeleteProject(storage, gmeAuth, projectName)
-            .then(function () {
-                return Q.allSettled([
-                    storage.closeDatabase(),
-                    gmeAuth.unload()
-                ]);
-            })
-            .nodeify(done);
+        Q.allSettled([
+            storage.closeDatabase(),
+            gmeAuth.unload()
+        ])
+        .nodeify(done);
```

Comment:
this will lose any error raised by closeDatabase or unload

e.g.
    > q.allSettled([q.reject('asdf')]).nodeify(function (err, res) { console.log(err + ': ' + res) })
    undefined
    > null: [object Object]